src/check.py

```diff
@@ -403,14 +403,16 @@ def check_answer_contains_points(doc:Document,start:int,end:int):
         
         if "記述設問" in question:
             if "解答のポイント" not in question:
```

Comment:
elseなしで、ifが連続するならそれって実質的にand条件だと思います。
また、ここのif文によってforループのフィルタリングをしているようなので、


```suggestion
    for question in question_explanation_list if ("記述設問" in question) and not ("解答のポイント" in question)

```

のように書けるのではないっでしょうか？